src/main/cpp/hw/DragonTalon.cpp

```diff
@@ -309,6 +309,14 @@ shared_ptr<MotorController> DragonTalon::GetSpeedController() const
 	return m_talon;
 }
 
+void DragonTalon::SetIntegratedSensorPosition(double newPos, double timeoutMs) const
+{
+	if (m_talon != nullptr)
+	{
+		m_talon.get()->GetSensorCollection().SetQuadraturePosition(newPos, timeoutMs);

```

Comment:
really need to look that 
A - there is a sensor
B - what type of sensor is it

Most common case will be a quadencoder, but it isn't a given